src/test/scala/com/wavesplatform/state/diffs/smart/predef/SerContextFunctionsTest.scala

```diff
@@ -80,105 +80,104 @@ class SerContextFunctionsTest extends PropSpec with PropertyChecks with Matchers
       0, 0, 26, 1, 84, 100, 23, -115, -33, -128, -20, -97, 62, -33, -42, 86, -24, -22, 104, -110, -85, 40, -23, -25, 122, -18, -70, -100, -99, 7, 4,
       0, 0, 0, 5, 98, 97, 115, 105, 99, 3, 3, 3, 5, 0, 0, 0, 7, 108, 111, 110, 103, 65, 108, 108, 5, 0, 0, 0, 9, 115, 117, 109, 83, 116, 114, 105,
       110, 103, 7, 5, 0, 0, 0, 13, 115, 117, 109, 66, 121, 116, 101, 86, 101, 99, 116, 111, 114, 7, 5, 0, 0, 0, 7, 101, 113, 85, 110, 105, 111, 110,
-      7, 4, 0, 0, 0, 6, 110, 101, 80, 114, 105, 109, 3, 3, 9, 1, 0, 0, 0, 2, 33, 61, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 3, -24, 0, 0, 0, 0, 0, 0, 0, 3,
+      7, 4, 0, 0, 0, 6, 110, 101, 80, 114, 105, 109, 3, 9, 1, 0, 0, 0, 2, 33, 61, 0, 0, 0, 2, 0, 0, 0, 0, 0, 0, 0, 3, -24, 0, 0, 0, 0, 0, 0, 0, 3,
```

Comment:
Wait, the same script should restult in the same bytes, and what's more important, the old bytes should deser to the same script. Why has this changed?

Reply:
Accidently change the AllFunctionsScript.  Reverted to origin version.